test/integration/cdi/example-driver/kubeletplugin/server/noderegistrar.go

```diff
@@ -0,0 +1,134 @@
+package server
```

Comment:
When adding new files, remember to add the copyright header.

Reply:
done.!